### ISSUE TITLE
How do I use this??

### ISSUE THREAD

**Issue by akaready** [omkarmoghe/Clay-CSV#33]
I installed this extension but have no idea how to use it on a CSV file... there's no command under the CMD+SHIFT+P menu to run, nor anything under the menu when you right-click on the file name in the editor? Where else could I go to use this? It looks like exactly what I was looking for tonight... 🤔

**Comment by snehilshah** [comment]
Yup same, I really liked the features shown in the readme but I cannot find a way to use this

**Comment by deimonn** [comment]
Little late to the party, but you simply right-click a `.csv` or `.tsv` file, select "Open With..." and then click "CSV Table View".

Author may want to document this in the README, because I also struggled to find it for like 5 minutes.
